src/System.Management.Automation/security/SecuritySupport.cs

```diff
@@ -296,55 +359,14 @@ internal static ExecutionPolicy GetExecutionPolicy(string shellId, ExecutionPoli
                 case ExecutionPolicyScope.MachinePolicy:
                     {
                         string groupPolicyPreference = GetGroupPolicyValue(shellId, scope);
-                        if (!String.IsNullOrEmpty(groupPolicyPreference))
+                        // Be sure we aren't being called by Group Policy
```

Comment:
Our coding style puts a newline above comment blocks, as it represents a paragraph of thought.